[github-actions] script check fails frequently

[logs_6926.zip](https://github.com/openthread/ot-br-posix/files/6297333/logs_6926.zip)

This issue seems to happen a lot recently. 

```
2021-04-12T01:37:39.3157932Z otbr-agent[19207]: [adproxy] Stopped
2021-04-12T01:37:39.3162790Z otbr-agent[19207]: Initialize OpenThread Border Router Agent: OK
2021-04-12T01:37:39.3177631Z otbr-agent[19207]: Border router agent started.
2021-04-12T01:37:39.3182980Z otbr-agent[19207]: [INFO]-CORE----: Notifier: StateChanged (0x00038200) [NetData PanId NetName ExtPanId]
2021-04-12T01:37:39.3216581Z otbr-agent[19207]: [INFO]-PLAT----: Host netif is down
2021-04-12T01:37:39.3223194Z otbr-agent[19207]: [INFO]-BR------: infra interface state changed: RUNNING, link-local-addr=fe80:0:0:0:20d:3aff:fec5:f141
2021-04-12T01:37:39.3249345Z otbr-agent[19207]: [INFO]-BR------: infra interface state changed: RUNNING, link-local-addr=fe80:0:0:0:20d:3aff:fec5:f141
2021-04-12T01:37:44.2001546Z ##[error]Process completed with exit code 1.
```
`otbr-agent` just exit unexpectedly without further information.
Currently, I don't have any clue on this issue.

--- a/tests/scripts/check-scripts
+++ b/tests/scripts/check-scripts
@@ -72,7 +72,7 @@ main()
     SERVICES_PID=$!
     sudo service tayga start
     echo 'Waiting for services to be ready...'
-    sleep 10
+    sleep 30
     check_otbr_agent
     netstat -an | grep 80
     pidof tayga

